cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,122 @@
 /// <reference types='cypress' />
+const user = {
+  firstName: 'inna',
+  lastName: 'levytska',
+  email: 'inna123@gmail.com',
+  age: '40',
+  salary: '100',
+  department: 'QA'
+}
+beforeEach(() => {
+  cy.visit('https://demoqa.com/webtables')
+  });
 
 describe('Web Tables page', () => {
   it('', () => {
 
+
+  });
+  it('should have pagination', () => {
+    cy.get('.-pagination')
+      .should('contain', 'Previous')
+      .and('contain', 'Next');
+    cy.contains('.-pageInfo', 'Page')
+      .should('exist');
+  });
+  it('should have row count selection', () => {
```

Comment:
you do not check here nothing. You just select 50 rows. Add assertion that when you select 50 rows, table rows count changed to 50 rows

Reply:
not fixed